### ISSUE TITLE
Support Perl Language Server in vscode

### ISSUE THREAD

**Issue by alexgarel** [openfoodfacts/openfoodfacts-server#6505]
It would be nice to have support for Perl Language Server in vscode.

**Comment by alexgarel** [comment]
This PR https://github.com/openfoodfacts/openfoodfacts-server/pull/6491 adds percritic and there is a first work to add perl language server.

For perlcritic we have to manually patch extension we should
- [ ] propose a fix for https://github.com/sfodje/perlcritic/issues/26

However we are blocked by different issues for perl language server, in the configuration I imagined:
- https://github.com/richterger/Perl-LanguageServer/issues/131
- https://github.com/richterger/Perl-LanguageServer/issues/132

The author of the extension propose to go for ssh instead of launching through docker. This has the drawback that:
- we must add openssh to docker build
- you must be running your docker before vscode and independently (and we shall not use the main backend process for we restart it frequently)

Third option is to go for [vscode remote container](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) the drawback here is that we have to:
- duplicate all the installs done for backend in a new dockerfile (you have to start from vscode container to have all needed tools installed)
  - this maybe mitigated a bit by creating a dockerfile concatenating others, but still there will be duplication (eg. for package installs, which in turns could be delegated to reusable scripts, but this does augment complexity quite a lot)
- duplicate all the specific mounts / env vars settings that are used for backend and frontend
